.github/workflows/publish.yml

```diff
@@ -22,10 +22,6 @@ on:
         description: "Build folder in drafts to publish"
         type: string
         required: true
-      publishing_version:
-        description: "Data product release version"
-        type: string
-        required: true
```

Comment:
dropping this because it's better to infer it from metadata than force someone to always but the correct value here?

Reply:
Precisely